### PR TITLE
unit test

### PR DESCRIPTION
**Pull request by sunbufu**
Hi, this PR commit 2 files for unit test.
+ CoderUtilTest
+ RegistryServerSyncTest

### CODE REVIEW

File: dubbo-admin-backend/src/test/java/org/apache/dubbo/admin/common/util/CoderUtilTest.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
+package org.apache.dubbo.admin.common.util;
```
Comment (biyuhao) [inline dubbo-admin-backend/src/test/java/org/apache/dubbo/admin/common/util/CoderUtilTest.java]: Please add Apache License announcement at the head of new files.
You can copy it from any other sourcecode.

File: dubbo-admin-backend/src/test/java/org/apache/dubbo/admin/service/RegistryServerSyncTest.java [hunk 2]
```diff
@@ -0,0 +1,86 @@
+package org.apache.dubbo.admin.service;
```
Comment (biyuhao) [inline dubbo-admin-backend/src/test/java/org/apache/dubbo/admin/service/RegistryServerSyncTest.java]: Apache license announcement is required here.

File: dubbo-admin-backend/src/test/java/org/apache/dubbo/admin/common/util/CoderUtilTest.java [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
```
Comment (lovepoem) [inline dubbo-admin-backend/src/test/java/org/apache/dubbo/admin/common/util/CoderUtilTest.java]: remove *

File: dubbo-admin-backend/src/test/java/org/apache/dubbo/admin/common/util/CoderUtilTest.java (hunk 1 above)
Comment (sunbufu) [reply dubbo-admin-backend/src/test/java/org/apache/dubbo/admin/common/util/CoderUtilTest.java]: ok

File: dubbo-admin-backend/src/test/java/org/apache/dubbo/admin/service/RegistryServerSyncTest.java (hunk 2 above)
Comment (sunbufu) [reply dubbo-admin-backend/src/test/java/org/apache/dubbo/admin/service/RegistryServerSyncTest.java]: ok

File: dubbo-admin-backend/src/test/java/org/apache/dubbo/admin/common/util/CoderUtilTest.java (hunk 3 above)
Comment (sunbufu) [reply dubbo-admin-backend/src/test/java/org/apache/dubbo/admin/common/util/CoderUtilTest.java]: ok

**Pull request merged**